content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesHelperAction.java

```diff
@@ -171,6 +171,8 @@
 	/** Tool property to enable Drag and Drop uploads in a per-tool basis */
 	private static final String TOOL_PROP_DRAGNDROP_ENABLED = "content.upload.dragndrop";
 
+	private static final String LABEL_VERSION = "label.version";
```

Comment:
When you're only using this string in one place it isn't worth extracting out a constant for it. The class becomes more cluttered as a result. Sorry for not spotting this the first time around.